### DISCUSSION TITLE
BT-15 - BT-19 Felder

### DISCUSSION THREAD

**Discussion by UserDinDF** [stephanstapel/ZUGFeRD-csharp#392] [General]
Guten Tag Stephan. Danke für deine Rep. Ich bin auf der suche nach BT15 bis BT19. Im Projekt habe ich keinen passenden Klass gefunden, heisst es anders vielleicht?

Gesucht:
Wareneingangreferenz (BT-15) 
Lieferavisreferenz (BT-16) 
Vergabenummer (BT-17) 
Objektkennung (BT-18) 
Buchhaltungsreferenz (BT-19) 

Vielen Dank

**Comment by stephanstapel** [comment]
Das meiste ist vorhanden:

BT-16: InvoiceDescriptorDescriptor.DespatchAdviceReferencedDocument
BT-17/ BT-18: InvoiceDescriptorDescriptor.AdditionalReferencedDocuments
BT-19: InvoiceDescriptor.ReceivableSpecifiedTradeAccountingAccounts 

Tipp: Wenn Du die Schlüsselworter aus der Spezifikation im Quellcode suchst, findest Du die passenden Felder.
